CONTRIBUTING.md

```diff
@@ -13,62 +13,87 @@ The intent and goal of open sourcing this project is to increase the contributor
-- [x] Reviews
-  - Changes must be approved via peer code review
+-   [x] Clean, simple, well styled code
+-   [x] To ensure consistent commit messages, we enforce the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) style. After installing project dependencies with `npm install`, run `npx husky install` to set up automatic commit validation with `commitlint`.
+-   [x] Commits should be atomic and messages must be descriptive. Related issues should be mentioned by Issue number.
+-   [x] Comments
+    -   Module-level & function-level comments.
+    -   Comments on complex blocks of code or algorithms (include references to sources).
+-   [x] Tests
+    -   The test suite, if provided, must be complete and pass
+    -   Increase code coverage, not versa.
+    -   Use any of our testkits that contains a bunch of testing facilities you would need. For example: `import com.salesforce.op.test._` and borrow inspiration from existing tests.
+-   [x] Dependencies
+    -   Minimize number of dependencies.
+    -   Prefer Apache 2.0, BSD3, MIT, ISC and MPL licenses.
+-   [x] Reviews
+    -   Changes must be approved via peer code review
 
 # Creating a Pull Request
 
-1. **Ensure the bug/feature was not already reported** by searching on GitHub under Issues.  If none exists, create a new issue so that other contributors can keep track of what you are trying to add/fix and offer suggestions (or let you know if there is already an effort in progress).
-3. **Clone** the forked repo to your machine.
-4. **Create** a new branch to contain your work (e.g. `git br fix-issue-11`)
+1. **Ensure the bug/feature was not already reported** by searching on GitHub under Issues. If none exists, create a new issue so that other contributors can keep track of what you are trying to add/fix and offer suggestions (or let you know if there is already an effort in progress).
+2. **Clone** the forked repo to your machine.
+3. **Create** a new branch to contain your work (e.g. `git br fix-issue-11`)
 4. **Commit** changes to your own branch.
 5. **Push** your work back up to your fork. (e.g. `git push fix-issue-11`)
 6. **Submit** a Pull Request against the `main` branch and refer to the issue(s) you are fixing. Try not to pollute your pull request with unintended changes. Keep it simple and small.
 7. **Sign** the Salesforce CLA (you will be prompted to do so when submitting the Pull Request)
 
 > **NOTE**: Be sure to [sync your fork](https://help.github.com/articles/syncing-a-fork/) before making a pull request.
 
+# Release Process
+
+### Branching Strategy:
+
+-   **Main Branch**: Ongoing development and new features are submitted to the main branch.
+
+-   **Release Branch**: We'll utilize a dedicated branch for managing releases
+
+    -   **Major Release:**
+        -   Update the `.releaserc.yml` file: Locate the branches property and specify the newly created release branch name (e.g., `release-2`).
```

Comment:
should we switch line 82 with line 83? 82 line say "newly created brach ...",   83 line is for created the new branch ?